tests/test_ext.py

```diff
@@ -52,3 +52,41 @@ def test_types(T):
     if T is m.SInt:
         c = a.sext(ext)
         assert isinstance(c, T)
+
+
+def test_ext_n():
+    value = m.SInt[32]()
+
+    # zext TypeError.
+    with pytest.raises(TypeError) as pytest_e:
+        m.zext(value, None)
+        assert False
```

Comment:
The assert False is unnecessary in this pattern since `pytest.raises` will raise an exception if `TypeError` isn't raised

Reply:
Done.